### ISSUE TITLE
get_model() example fails

### ISSUE THREAD

**Issue by lemna** [jchrom/trelloR#22]
Hi,

the example for `get_model()` runs fine until this line
```r
> card1act = get_card_actions(card1id) # Get all actions performed on that card

GET URL:
https://api.trello.com/1/card/5ab955ae0be987052f7597e0/actions?limit=1000

  |======================================================================================================================| 100%
Fetched 0 result(s)

Request complete
Warning messages:
1: In min(model_ids) : no non-missing arguments to min; returning Inf
2: Binding failed: Argument 1 must be a data frame or a named atomic vector, not a response
returning list 
```
This is trelloR 0.4.0. Am I doing this wrong?

**Comment by jchrom** [comment]
Hi @lemna ,
thanks for letting me know. The few examples I managed to put in that would not require authentication make calls to Trello's public roadmap board, which might have changed in a way that makes it fail. That being said, it should probably fail more gracefully to let you know what happened, or I should come up with a better way to keep some example board up for trelloR.

I will look into this when I get the chance. In the meantime, just try using the vignette to get an idea of how things work, and if you encounter further issues, let me know.

**Comment by jchrom** [comment]
Yeah actually looking at your error message properly, it didn't fetch anything. So the following steps failed. I should definitely handle this better.

**Comment by lemna** [comment]
Wouldn't it be better to create a "trelloR" example board yourself? That way you have full control.

**Comment by jchrom** [comment]
I would have full control, true. But then I would have to worry about creating the content and keeping it up... but I might end up doing that.

I just noticed that example bit is also outdated because it contains mention of "paging" argument, which is deprecated. I'll try and fix all that as well.

**Comment by jchrom** [comment]
And one more comment, before I actually do anything: What you see is not a real fail, as in, an error. Instead, it just returned 0 results, because there are 0 actions happening on that card (like nobody adding any comment etc.). It just warns you about some other things which usually happen if some results are returned, but cannot happen now.

So to answer your question: No, you are not doing it wrong. There are just no actions associated with that card.

**Comment by jchrom** [comment]
Just released [v0.5](https://github.com/jchrom/trelloR/releases/tag/v0.5.0) - please upgrade if you can, and come back with any issues you find.
